src/source/main.cpp

```diff
@@ -152,24 +147,11 @@ int main(int argc, char** argv) {
 	try{
 #endif
 		hresult = OleInitialize(NULL);
-
```

Comment:
I think this commented out codeblock was some somewhat important code pertaining to trusted URLs

Reply:
I will store it elsewhere for the time being